src/Commands/Command.php

```diff
@@ -252,6 +259,16 @@ public function getName()
         return $this->name;
     }
 
+    /*
+     * Get Show in Help
+     *
+     * @return bool
+     */
+     public function showInHelp()
+     {
+     	return $this->show_in_help;
```

Comment:
Indent by 4 spaces, always.
What editor do you use? Make sure it has an [editorconfig](http://editorconfig.org/) plugin installed.